# Abstracts

## Day 1

### Shannon Schlueter (UNCC; “Universal data science challenges: Perspective from disparate data domains”):

Data science is largely a domain agnostic discipline. Best practices and novel solutions to once intractable challenges are routinely gleaned from seemingly disparate data domains. Methodologies once rarely identified outside of the realm of bioinformatics and computational biology are now commonly used to address challenges in fields such as image recognition and social sentiment analysis. But is this synergy mutually beneficial? Can the life sciences and bioinformatics claim benefit from practices found in other data rich domains?

### Charles Unsworth (Auckland University; “Machine learning for neural applications”)

In the first part of the seminar, I will discuss how we have applied ANNs for the detection of volatile organic compounds (VOCs) from the in vivo firings of insect olfactory neurons. In the second part of the seminar, I will discuss how we have applied Machine Learning approaches to identify Sharp wave transients in the EEG in order to aid the early detection of Hypoxic Ischemia in Newborn Infants.

### Richard Newcomb (Chief Scientist, Plant & Food Research; “Data to do with Smell”).

Smell is one of the most complex of our senses. Thousands of odours, hundreds of receptor genes, a few sensory perceptions and all with just one nose. In this talk I outline how we have used analysis and visualisation of data from genomes and odour space to unravel how odours are perceived in insects and humans

### Tatiana Lomasko (CSST; “Big Space Data: What you do with it?”)

The focus of this seminar is presenting the opportunities rasising from the establshment of the NZ Centre of Space Science Technology (CSST). New Zealand needs to capitalise on the rapidly developing global opportunity provided by satellites, both for Earth observation and for data telemetry. This opportunity is relevant for many sectors of New Zealand's economy and society including primary production, energy and water management, national security, urban planning and hazard management. I will present the CSST’s plans to profit from extracting the full value available from existing space-based measurements that are often made freely available by international space agencies as well as from developing a national capability to own and operate our own satellites to enhance our country's wealth and well-being.

There is exponential growth in the volume of satellite data, primarily imagery, being made available. This "Earth Observation" market was estimated at NZ$2.7 billion in 2015 (by North-ern Sky Research) and is predicted to double by 2024. The market segment comprises data, data processing, and information products, with value-added information products undergoing the most rapid growth and having the greatest future potential.

### Finlay Thompson (DragonFly; "Data science practices in the real world”)

While doing research in a reproducible way is recognised as desirable goal, this ambition can sometimes be neglected when confronted with the heat of tight deadlines and busy research programmes. At Dragonfly we have developed/discovered a set of practices that ensures our work is always reproducible. I will talk about what those principles are, and how committing to reproducible research has increased the reliability of our work, and the productivity of our team.

### Bruno Lago (Catalyst): Cloud Computing for Data Scientists

Cloud Computing is a disruptive technology, but only when done right. In this presentation Bruno will explain how data scientists and researchers can benefit from cloud computing, describe options to store and process massive amounts of data, and what software testing methods can be used to ensure the integrity of the code, process and results. 

### Mingshu Cao (AgResearch): Transforming Big Data into Big Insights – a Journey of Endless Paradigm Shifting?

Driven by modern instrumentation, big data in Omics has brought many opportunities and the fundamental changes. The changes include the approaches we store, analyse and share data; conclusions we can effectively reach and validate; and even in the way we communicate complex information to stakeholders. Based on my experience working with large data in Metabolomics, Transcriptomics and Genomics I present case studies to demonstrate the power of big data in generating exciting leads and the challenges we need to face. Making better use of big data in Omics demands enhanced computing environment, capability of dealing with heterogeneous information, and novel algorithms to reveal new connections and to model system effects of complex traits.    

### John McCallum (PFR): Better not Bigger: Thoughts on Foundation Data Science in Biology 

Few biologists handle 'big data’ (> 1Tb) but now frequently encounter ‘large data’ that stretches their computational toolset and skills. There are often financial, social, technological or business barriers to data aspirants wishing to scale their practice. Realising value from data-rich technologies for our lives, business and communities will require that we work together to raise standards of digital scholarship, support diversity, and adopt lifelong learning practices. This presentation will provide an opinionated review of experience gained with large data and highlight the foundation data science skills that are helping us. In particular we will demonstrate examples of how the Python ecosystem is enabling choice, collaboration and scalability.  

### Sara Jaeger (PFR): Analysing Twitter and Unstructured Text Data

### Peter McAtee (PFR): KiwiScan

Selective breeding in fruit trees, arable crops and finfish currently involves raising populations of tens of thousands of individuals obtained from multiple controlled crosses of elite parents. Such large populations mean that during the early stages of selection (e.g. “stage 1” in PFR’s pip- and kiwifruit breeding programmes) only a subset of individuals are evaluated based on multiple traits. Phenomic data is used for calculating the individuals’ breeding potential and for the breeders to make decisions about which individuals to move to further stages or cull.  Unfortunately, the number of individuals that can be phenotypically assessed is limited and a large proportion of raised individuals often get deemed non-suitable for evaluation in the field.  As a consequence, the phenomic information from these individuals is not captured, weakening the knowledge gained from our programs. Phenotypic assessment of all individuals is a time-consuming process, subject to human error, is currently not reproducible, and uses progressive thresholds.

KiwiScan is a platform that was developed to address some of these issues. It was developed in-house to collect, storage, and computationally assess cross-sectional images of kiwifruit. This platform utilises open source image analysis software (ImageJ) to estimate tissue boundaries and then quantify the amount of colour present within each tissue selection. The use of this quantification platform  in conjunction with a previously genotyped kiwifruit block  has validated the selection of key red QTL locations and  highlighted some potential candidate QTL's for other important breeding traits.

### Ben Pyluc Jolly, David Medyckyj-Scott, Raphael Spiekermann, and Anne-Gaelle Ausseil (Landcare): A Tool for the Repeatable Generation and Automated Documentation of Land-use Classification Maps

### Farhana Pinu (PFR): First Draft Metabolic Reconstruction of a Wine Yeast

### Burgueño, Eric (PFR): Making Science Better: A Deep Dive into powerPlant


